src/openrct2-ui/windows/Map.cpp

```diff
@@ -69,19 +69,30 @@ namespace OpenRCT2::Ui::Windows
         return getTechnicalMapSize() * 2;
     }
 
+    static bool isEditorOrSandbox()
+    {
+        return (gScreenFlags & SCREEN_FLAGS_SCENARIO_EDITOR) || GetGameState().Cheats.SandboxMode;
+    }
```

Comment:
Not for this PR, but I think we should move the screen flags and scene logic to a 'scene manager' of sorts, and replace exposing the `gScreenFlags` flags with a function like this.